libraries/AP_Airspeed/AP_Airspeed_SDP3X.cpp

```diff
@@ -0,0 +1,303 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+  backend driver for airspeed from a I2C SDP3X sensor
+
+  with thanks to https://github.com/PX4/Firmware/blob/master/src/drivers/sdp3x_airspeed
+ */
+#include "AP_Airspeed_SDP3X.h"
+
+#include <stdio.h>
+
+#define SDP3X_SCALE_TEMPERATURE		200.0f
+
+#define SDP3XD0_I2C_ADDR            	0x21
+#define SDP3XD1_I2C_ADDR            	0x22
+#define SDP3XD2_I2C_ADDR            	0x23
+
+#define SDP3X_CONT_MEAS_AVG_MODE    	0x3615
+#define SDP3X_CONT_MEAS_STOP        	0x3FF9
+
+#define SDP3X_SCALE_PRESSURE_SDP31	60
+#define SDP3X_SCALE_PRESSURE_SDP32	240
+#define SDP3X_SCALE_PRESSURE_SDP33	20
+
+#define CONSTANTS_AIR_DENSITY_SEA_LEVEL_15C             1.225f                  /* kg/m^3               */
```

Comment:
AP_Baro hard coded this one. https://github.com/ArduPilot/ardupilot/blob/f2b7dc74d7bfbc9004324615ccac085999f1b141/libraries/AP_Baro/AP_Baro.cpp#L282

Reply:
fixed